src/libraries/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -6,12 +6,9 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetsWindows)' != 'true'">
     <GeneratePlatformNotSupportedAssemblyMessage>SR.PlatformNotSupported_System_Windows_Extensions</GeneratePlatformNotSupportedAssemblyMessage>
-    <GeneratePlatformNotSupportedAdditionalParameters>--exclude-api-list ExcludeApiList.PNSE.txt</GeneratePlatformNotSupportedAdditionalParameters>
   </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="TypeForwards.cs" />
-  </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <Compile Include="TypeForwards.cs" />
```

Comment:
The typeforwards need to be on both code paths right? So that the generate reference source doesn't generate a PNSE for those?

Reply:
> The typeforwards need to be on both code paths right?

Correct. It will be taken from ref in case of geneartePlatformNotSupported case.

> So that the generate reference source doesn't generate a PNSE for those?

generate not supported assembly will not be doing anything to typeforwards, will just copy them to the intermediate folder and compile.